CHANGELOG.md

```diff
@@ -19,52 +19,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-17
 
+Amazon Chime SDK React Components Library v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK React Components Library v3 includes major improvements for component style customization, `MeetingProvider/MeetingManager` configuration, device management, WebRTC metrics and logger.
+
+- **Component style customization**: Improve components for easier customization of style. Publish this [styling guide](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/styling-guide--page) to instruct on how to customize the component style.
```

Comment:
Suggest to update:
"Publish..." sentence to just "Checkout our [styling guide](link-to-styling-guide) for more information".

Reply:
Updated!